pd/src/main.rs

```diff
@@ -57,11 +61,16 @@ enum RootCommand {
         /// Bind the gRPC server to this port.
         #[clap(short, long, default_value = "8080")]
         grpc_port: u16,
-        /// Bind the metrics endpoint to this port.
+        /// bind the metrics endpoint to this port.
         #[clap(short, long, default_value = "9000")]
         metrics_port: u16,
+        /// bind the tendermint proxy server to this port.
+        #[clap(short, long, default_value = "36657")]
+        tendermint_proxy_port: u16,
```

Comment:
do we need a separate port for this? ideally we'd just add it int o the main GRPC port and let the router forward requests appropriately, since the goal is to have just one endpoint.